src/lib/forms/rjsf/widgets/RJSFTimeWidget.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { TimeInput } from 'semantic-ui-calendar-react';
+import { Form } from 'semantic-ui-react';
+
+/**
+ * Custom widget for a time field
+ */
+export class RJSFTimeWidget extends Component {
+  render() {
+    const {
+      id,
+      value,
+      required,
+      disabled,
+      placeholder,
+      readonly,
+      label,
+      onChange,
+    } = this.props;
+    return (
+      <Form.Field
+        id={id}
+        key={id}
+        required={required}
+        disabled={disabled || readonly}
```

Comment:
Question: `readonly` is not used anywhere if I am not mistaken. Do we have it to improve readability in the future?

Reply:
I am not sure what you mean exactly with `improve readability`, but this is part of the API contract of the RJSF widgets APIs, so if we use it in a schema, it is already working correctly.